caboose.go

```diff
@@ -101,6 +108,8 @@ const DefaultFetchKeyCoolDownDuration = 1 * time.Minute // how long will a sane
 // however, only upto a certain max number of cool-offs.
 const DefaultSaturnNodeCoolOff = 5 * time.Minute
 
+const DefaultSentinelCidPeriod = int64(200)
```

Comment:
comment needed for what this is, because it isn't really a `Period` - this means that sentinel will be used once every 200 requests, on average, is how i think i'm interrpeting this?

Reply:
Your interpretation is correct, added a comment to clarify the value.